package.json

```diff
@@ -37,7 +37,6 @@
     "jest": "^29.4.2",
     "jest-environment-jsdom": "^29.4.1",
     "jest-junit": "^9.0.0",
-    "jsdoc": "^4.0.0",
```

Comment:
I don't think `npm run docs` will work anymore since it relies on this package. Is there anyway we can keep it but just less feature-ful after the cleanup you did? Otherwise, i'm not against removing it for now. It would be good to revive auto doc generation eventually but doesn't need to block Typescript in my opinion as this feels like more valuable "documentation" anyway. 

If you do remove it you'll also have to remove the script from `bin/` and the link in the README.

Reply:
I removed it to align with the same change made in unchained-bitcoin. The jsdocs comments have been gutted and are now basically just descriptions.

Reply:
Yeah but this dependency is used in a script that is still here. So if we keep the script to generate docs even if it’s just descriptions then we need this dependency I think.

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff